static/scss/faculty-members.scss

```diff
@@ -0,0 +1,111 @@
+// sass-lint:disable mixins-before-declarations
```

Comment:
Given the amount of styles in `product-detail.scss`, I think it's fine to define these in a separate scss file, but I think we'd benefit from naming this `product-faculty-members.scss` to at least give a clearer indication that this is part of the product detail page